CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Skipping redraws on PTY update
 - Not redrawing while resizing on Windows/macOS
 - Decorations `none` launching an invisible window
+- Unfocused hollow cursor until refocus on some systems
```

Comment:
You've said this doesn't happen on master, right? Then we don't need a change log entry.

Reply:
There is no way to launch use non SRGB buffer on Windows, so this issue should be presented on master in my opinion, I just can't repro due to other bug.

Reply:
If there's no way to launch them, the bug doesn't exist. This is not an issue on the other platforms.

So as far as I understand it, there should be no way to run into this on master?

Reply:
Ok, I've simulated the behavior on X11 and Wayland and wont able to repro. There weren't any Focused(false) on window close.

Reply:
* on master branch *

Reply:
Yeah, I can confirm that this isn't an issue on macOS and X11. So if you've tested Wayland too, I think there's no reason to add a change log entry. Not like we're short on change log entries for the next release. :D